homeassistant/components/reolink/strings.json

```diff
@@ -86,73 +86,153 @@
   "entity": {
     "binary_sensor": {
       "face": {
-        "name": "Face"
+        "name": "Face",
+        "state": {
+          "off": "[%key:component::binary_sensor::entity_component::gas::state::off%]",
+          "on": "[%key:component::binary_sensor::entity_component::gas::state::on%]"
+        }
       },
       "person": {
-        "name": "Person"
+        "name": "Person",
+        "state": {
+          "off": "[%key:component::binary_sensor::entity_component::gas::state::off%]",
+          "on": "[%key:component::binary_sensor::entity_component::gas::state::on%]"
+        }
       },
       "vehicle": {
-        "name": "Vehicle"
+        "name": "Vehicle",
+        "state": {
+          "off": "[%key:component::binary_sensor::entity_component::gas::state::off%]",
+          "on": "[%key:component::binary_sensor::entity_component::gas::state::on%]"
+        }
       },
       "pet": {
-        "name": "Pet"
+        "name": "Pet",
+        "state": {
+          "off": "[%key:component::binary_sensor::entity_component::gas::state::off%]",
+          "on": "[%key:component::binary_sensor::entity_component::gas::state::on%]"
+        }
       },
       "animal": {
-        "name": "Animal"
+        "name": "Animal",
+        "state": {
+          "off": "[%key:component::binary_sensor::entity_component::gas::state::off%]",
+          "on": "[%key:component::binary_sensor::entity_component::gas::state::on%]"
+        }
       },
       "visitor": {
         "name": "Visitor"
```

Comment:
No visitor detection?

Reply:
Well that is when the doorbell is beeing pressed.
I thought "detected" would be a bit weird.
I know that "visitor" is weird for a doorbell press, but that is what reolink uses in their app so I kept it in line with their app.

I could not think of a nice way to translate that so thougth to stick to on/off.

What do you think?

Reply:
That makes sense!